IPython/lib/pretty.py

```diff
@@ -918,7 +918,7 @@ def _counter_pprint(obj, p, cycle):
     if cycle:
         p.pretty(cls_ctor(RawText("...")))
     elif len(obj):
-        p.pretty(cls_ctor(dict(obj)))
+        p.pretty(cls_ctor(dict(sorted(obj.items(), key=lambda x: x[1], reverse=True))))
```

Comment:
Ho ! That was an easy update ! 

Isn't `sorted(obj.items(), key=lambda x: x[1], reverse=True)` simply `obj.most_common()` ? which might be simpler or more readable ?

```suggestion
        p.pretty(cls_ctor(dict(obj.most_common())))
```

I'm guessing this will also get rig of the linting failure that is likely to complain line is too long.

Reply:
Great suggestion (good one for me to know for the future as well)! 

I appreciate you looking at it. I've just added that change and re-run the tests and it appears to all be working still.